index.html

```diff
@@ -163,6 +163,7 @@ <h3>Editor</h3>
     <li><a href="https://github.com/editorconfig/editorconfig-geany#readme"><img src="logos/geany.png" alt="Geany"><span>Geany</span></a></li>
     <li><a href="https://github.com/editorconfig/editorconfig-gedit#readme"><img src="logos/gedit.png" alt="Gedit"><span>Gedit</span></a></li>
     <li><a href="https://github.com/editorconfig/editorconfig-jedit#readme"><img src="logos/jedit.png" alt="jEdit"><span>jEdit</span></a></li>
+    <li><a href=" https://github.com/skalogryz/editorConfig/tree/master/lazideext"><img src="logos/lazarus.png" alt="Lazarus"><span>Lazarus</span></a></li>
```

Comment:
You have an extra space before "https"

Reply:
Oh... 😱  
Deleted unnecessary white space 🙏